lib/g_searcher_web/controllers/dashboard_controller.ex

```diff
@@ -1,11 +1,13 @@
 defmodule GSearcherWeb.DashboardController do
   use GSearcherWeb, :controller
 
+  alias GSearcher.Reports
   alias GSearcher.SearchResults.Report
 
-  def index(conn, _params) do
-    report_changeset = Report.create_changeset(%{})
+  def index(conn, params) do
+    create_report = params[:create_report] || Report.create_changeset(%{})
```

Comment:
I am wondering why we need this and can we pass params directly to changeset 🤔 ?

Reply:
This is when we're re-rendering the page to display the error, we would want the `Ecto.Changeset` to be rendered instead of the `Report.create_changeset` 🤔